scripts/bash/run_enceladus.sh

```diff
@@ -197,7 +197,11 @@ case $key in
     shift 2 # past argument and value
     ;;
   --null-value)
-    NULL_VALUE="$2"
+    if [[ -z "${2// }" ]]; then
```

Comment:
Sorry, how this exactly works?
Not sure I understand the slashes there. And it's different from `--delimiter`. Shouldn't it be the same?

Reply:
The delimiter is a char, null-value is a string. So delimiter caters for a single char being a white space. This needs to cater for a whole string being a whitespace.

Reply:
As to why and what - https://www.gnu.org/savannah-checkouts/gnu/bash/manual/bash.html#Shell-Parameter-Expansion